x-pack/plugins/threat_intelligence/public/modules/indicators/components/query_bar/query_bar.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I'm not sure if I'm doing anything wrong (just running the `node scripts/storybook threat_intelligence` command) but I'm not seeing this in Storybook at all
<img width="238" alt="Screen Shot 2022-07-27 at 4 00 45 PM" src="https://user-images.githubusercontent.com/17276605/181266695-4abb154c-d387-470f-aa4c-d34e593d2fe3.png">

Reply:
It's back where it should be! :D 
![image](https://user-images.githubusercontent.com/11671118/181434616-d0f197a5-3127-40a3-b169-817afc741381.png)